### PR TITLE
added light theme to beautymail

### PR DESCRIPTION
**Pull request by elliotforbes**

### CODE REVIEW

**Comment by phroggyy** [comment]
Hey @emforce, would you be able to provide a screenshot of the template as well? :)

**Comment by emforce** [comment]
light

@phroggyy Added! :)

**Comment by phroggyy** [comment]
Looks like you've got a bit of unnecessary files there @emforce! Could you remove the sass cache and the `.css.map`, and then squash the commits?

**Comment by emforce** [comment]
I've removed the files and squashed everything into one commit @phroggyy

File: src/views/templates/light.blade.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<html>
+    <head>
+		<title>{{ $senderName or '' }}</title>
```
Comment (Rpsl) [inline src/views/templates/light.blade.php]: tabs

**Pull request closed without merging**